scripts/generateOptions.js

```diff
@@ -0,0 +1,17 @@
+//eslint-disable-next-line node/no-unpublished-require
+const { sync } = require('execa');
+const { resolve } = require('path');
+const { writeFileSync } = require('fs');
+
+const { stdout } = sync(resolve(__dirname, '../packages/webpack-cli/bin/cli.js'), ['--help=verbose'], {
+    cwd: __dirname,
+    reject: false,
+});
+
+// format output for markdown
+const mdContent = ['```bash', stdout, '```'].join('');
+
```

Comment:
need to wrap between ``` ```` because otherwise, it's not displaying it correctly in markdown.